deps.edn

```diff
@@ -8,6 +8,11 @@
         org.clojars.liverm0r/dartclojure {:mvn/version "0.2.23-SNAPSHOT"}
         vvvvalvalval/supdate {:mvn/version "0.2.3"}
         camel-snake-kebab/camel-snake-kebab {:mvn/version "0.4.3"}
+        no.cjohansen/replicant {:git/url "https://github.com/cjohansen/replicant.git"
+                                :git/sha "ef2fecbe301cafa8449b74a1e2cbe4fc3ddc18ac"}
```

Comment:
We should be using the latest stable Replicant, I think:

`:mvn/version "2025.03.27"`

Reply:
Ah yes, I didn't think to upgrade. This PR's been a long time in the making. 😄